MachineLearning_Engine/Compute/LinearRegression.cs

```diff
@@ -39,20 +39,22 @@ public static partial class Compute
         /*************************************/
 
         [Description("Finds the slope and the intercept of a linear function that best fits the given set of bidimensional data.")]
-        [Input("x", "Training data as a list of 2-elements list")]
-        [Input("y", "Target values as a list of 2-elements list")]
-        [Output("model", "The ordinary least squares linear regression with the given data.")]
-        public static LinearRegression LinearRegression(Tensor x, Tensor y)
+        [Input("x", "Training data as a list of 2-elements list.")]
+        [Input("y", "Target values as a list of 2-elements list.")]
+        [MultiOutput(0, "model", "The ordinary least squares linear regression with the given data.")]
+        [MultiOutput(1, "r2", "The coefficient of determination R^2 of the prediction.")]
+        public static Output<LinearRegression, Tensor> LinearRegression(Tensor x, Tensor y)
         {
             PyObject model = BH.Engine.MachineLearning.Compute.Invoke("LinearRegression.fit", x, y);
-            return new LinearRegression(model);
+            Tensor score = new Tensor(BH.Engine.MachineLearning.Compute.Invoke("LinearRegression.score", model, x, y));
+            return new Output<LinearRegression, Tensor> { Item1 = new LinearRegression(model), Item2 = score };
```

Comment:
You can reflecting this using a `Query.Error` method that takes in: the model, the unvisited Xs and the targets (Ys).